build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -171,7 +171,9 @@ public static BuildTargetResult CompileCoreHost(BuildTargetContext c)
         {
             var hostVersion = c.BuildContext.Get<HostVersion>("HostVersion");
             var configuration = c.BuildContext.Get<string>("Configuration");
-            string rid = c.BuildContext.Get<string>("TargetRID");
+            string rid = DotNetCli.Stage0.GetRuntimeId();
```

Comment:
>I think the fix should be removing the check at 

I put in that check so that the other build products would be put under the win7 folders like they were before my changes. If I remove that check, everything will be made win10-* on a win10 machine.

If we want that then that's fine, but I was under the impression that we wanted the packages to have win7-* RIDS.